Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def bump_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `bump` [<options>]
+
+        Display out-of-date brew formulae and the latest version available.
+        Also displays whether a pull request has been opened with the URL.
+      EOS
+      flag "--formula=",
+           description: "Return results for package by name."
+      flag "--limit=",
+           description: "Limit number of package results returned."
+      switch :verbose
+      switch :debug
+    end
+  end
+
+  def bump
+    args = bump_args.parse
+
+    requested_formula = args.formula
+    requested_limit = args.limit ? args.limit.to_i : nil
+    requested_formula&.downcase!
```

Comment:
In what situation would this not be downcased already? All formulae names are, I believe.